### PR TITLE
Fix: Fixes version of `sensiolabs-de/storyblok-bundle`

### PR DESCRIPTION
**Pull request by silasjoisten**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     | https://packagist.org/packages/sensiolabs-de/storyblok-bundle

### CODE REVIEW

**Review by OskarStark — DISMISSED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by silasjoisten** [comment]
I release initially the 1.0 version but i figured out thats a bad idea for an yet unstable bundle so i deleted the tags and released 0.1

File: sensiolabs-de/storyblok-bundle/0.1/config/packages/storyblok.yaml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 storyblok:
     base_uri: '%env(STORYBLOK_API_BASE_URI)%'
     token: '%env(STORYBLOK_API_TOKEN)%'
+#   assets_token: '%env(STORYBLOK_ASSETS_API_TOKEN)%'
```
Comment (OskarStark) [inline sensiolabs-de/storyblok-bundle/0.1/config/packages/storyblok.yaml]: Please add newline at the end of the file

File: sensiolabs-de/storyblok-bundle/0.2/config/routes/storyblok.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#storyblok:
+#    resource: '@StoryblokBundle/config/routes.php'
```
Comment (OskarStark) [inline sensiolabs-de/storyblok-bundle/0.2/config/routes/storyblok.yaml:2]: I would not comment it, the route doesn't hurt, if you a) don't add the route at Storyblok or b) don't create a handler for and event

Comment (silasjoisten) [inline sensiolabs-de/storyblok-bundle/0.2/config/routes/storyblok.yaml:2]:
```suggestion
storyblok:
    resource: '@StoryblokBundle/config/routes.php'
```

**Review by OskarStark — APPROVED** [review]

**Pull request merged**